### ISSUE TITLE
Error compilation Rust 1.7

### ISSUE THREAD

**Issue by fylux** [justinsacbibit/pong-rs#1]
![screenshot - 190316 - 20 27 50](https://cloud.githubusercontent.com/assets/1207962/13900897/52698762-ee11-11e5-8cd6-3ae8148c6922.png)

**Comment by justinsacbibit** [comment]
Hey @Flash-back, looks like there were some breaking changes in Rust 1.7 which breaks some of the dependencies. If you really want to run this project, I suggest using an earlier version of Rust.

**Comment by fylux** [comment]
Yes @justinsacbibit I see, I was posting if you were interested in updating the code to make it works with current Rust version.

Greetings.
